auth-web/src/models/AutoComplete.ts

```diff
@@ -0,0 +1,11 @@
+// To handle Von autocomplete responses
+export interface AutoCompleteResponseIF {
+    total: number,
+    results: Array<AutoCompleteResultIF>
+}
+
+export interface AutoCompleteResultIF {
```

Comment:
now I know IF is for interfaces!! it kind of read like an if statement so far...
Can you check if the prefix/postfix to denote interface is a good pattern? I think not..may be worth a discussion

Reply:
Latest trend to include IF to imply the response may be occuring from API call. But yeah, I shall refactor to reflect the existing best practices